client/web-sveltekit/src/lib/repo/FileHeader.svelte

```diff
@@ -26,10 +27,18 @@
             }
         ),
     ])
+
+    // HACK: we use a flexbox for the path and an inline icon, but we still want the copied path to be usable.
+    // This event handler removes the newlines surrounding slashes from the copied text.
+    function stripSpaces(event: ClipboardEvent) {
+        const selection = document.getSelection() ?? ''
+        event.clipboardData?.setData('text/plain', selection?.toString()?.replaceAll(/\n?\/\n?/g, '/'))
```

Comment:
`selection` should always be defined, since you do `?? ''` above. So I think you can drop the `?` here:

```suggestion
        event.clipboardData?.setData('text/plain', selection.toString().replaceAll(/\n?\/\n?/g, '/'))
```